larray/tests/generate_data.py

```diff
@@ -0,0 +1,182 @@
+             'missing_values': "a=1..3; b=b0,b1; c=c0..c2",
+             'unsorted': "a=3..1; b=b1,b0; c=c2..c0",
+             'position': "a=1..3; b=b0,b1; c=c0..c2"}
+
+    wb = open_excel(os.path.join(DATA_DIR, 'test.xlsx'), overwrite_file=True)
+    wb_narrow = open_excel(os.path.join(DATA_DIR, 'test_narrow.xlsx'), overwrite_file=True)
+
+    for name, dim in tests.items():
+        arr = ndtest(dim)
+        if name == '2d_classic':
+            df = arr.to_frame(fold_last_axis_name=False)
+            # wide format
+            df.to_csv(os.path.join(DATA_DIR, 'test{}.csv'.format(name)), sep=',', na_rep='')
+            wb[name] = ''
+            wb[name]['A1'].options().value = df
+            # narrow format
+            df = arr.to_series(name='value')
+            df.to_csv(os.path.join(DATA_DIR, 'test{}_narrow.csv'.format(name)), sep=',', na_rep='', header=True)
+            wb_narrow[name] = ''
+            wb_narrow[name]['A1'].options().value = df
+        elif name == 'missing_values':
+            df = arr.to_frame(fold_last_axis_name=True)
+            # wide format
+            df = df.drop([(2, 'b0'), (3, 'b1')])
+            df.to_csv(os.path.join(DATA_DIR, 'test{}.csv'.format(name)), sep=',', na_rep='')
+            wb[name] = ''
+            wb[name]['A1'].options().value = df
+            # narrow format
+            df = arr.to_series(name='value')
+            df = df.drop([(2, 'b0'), (2, 'b1', 'c1'), (3, 'b1')])
+            df.to_csv(os.path.join(DATA_DIR, 'test{}_narrow.csv'.format(name)), sep=',', na_rep='', header=True)
+            wb_narrow[name] = ''
+            wb_narrow[name]['A1'].options().value = df
+        elif name == 'position':
+            # wide format
+            wb[name] = ''
+            wb[name]['D3'] = arr.dump()
+            # narrow format
+            wb_narrow[name] = ''
+            wb_narrow[name]['D3'] = arr.dump(wide=False)
+        else:
+            # wide format
+            arr.to_csv(os.path.join(DATA_DIR, 'test{}.csv'.format(name)))
+            wb[name] = arr.dump()
+            # narrow format
+            arr.to_csv(os.path.join(DATA_DIR, 'test{}_narrow.csv'.format(name)), wide=False)
+            wb_narrow[name] = arr.dump(wide=False)
+
+    wb.save()
+    wb.close()
+    wb_narrow.save()
+    wb_narrow.close()
+
+
+def generate_example_files(CSV=True, Excel=True, HDF5=True):
+    from larray_eurostat import eurostat_get
+
+    def prepare_eurostat_data(dataset_name):
+        arr = eurostat_get(dataset_name)[X.unit['NR'], X.age['TOTAL'], X.sex['M,F']]
+        arr = arr[X.time[::-1]][2013:2017]
+        arr = arr.rename('sex', 'gender')
+        arr = arr.set_labels(gender='Male,Female')
+        arr = arr.rename('geo', 'country')
+        if dataset_name == 'migr_imm1ctz':
+            # example of an array with ambiguous axes
+            arr = arr['COMPLET', X.citizen['BE,LU,NL'], X.country['BE,LU,NL']].astype(int)
+            arr = arr.rename('citizen', 'nationality')
```

Comment:
I don't think this is correct! 'Belgium' is **not** a nationality, it is a country!